cargo-dist/src/tasks.rs

```diff
@@ -1393,6 +1415,30 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         checksum_idx
     }
 
+    fn add_updater(&mut self, release_idx: ReleaseIdx, variant_idx: ReleaseVariantIdx) {
+        if !self.local_artifacts_enabled() {
+            return;
+        }
+
+        let release = self.release(release_idx);
+        let variant = self.variant(variant_idx);
+        let filename = format!("{}-{}-update", release.app_name, variant.target);
```

Comment:
can't this just be `{variant.id}-{update}`?

Reply:
also is there a reason we need to generate N copies of this? We could just make one universal/shared one and the installer could just rename it on fetch..?

(tbh i expected the design here to be putting it inside the archives... hmm... interesting)

Reply:
Keeping it separate makes it easier to avoid fetching and installing it in the Homebrew and npm installers; that way there's no risk of it leaking into inappropriate places.

> We could just make one universal/shared one and the installer could just rename it on fetch..?

The updater is native code; we need one per target. We choose which version to fetch and install in the installers.